egs2/dns_icassp22/enh1/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+sample_rate=16k
```

Comment:
I think [DNS4](https://arxiv.org/abs/2202.13288) is designed to use fullband audio (48 kHz), right?

Reply:
Sure, but I'm not sure if we should use fullband, as it would increase the overhead (larger conv kernels, or longer feature sequence, etc.).

Reply:
Yes, the computation is increased in this case. However, to make our models comparable to the systems in DNS4 Challenge, we should try to use the same data by default.
Meanwhile, it can be the user's choice to use a lower sample rate.